cmd/weirproxy/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 	}
 
 	configFile := rootCmd.PersistentFlags().String("config", "conf/weirproxy.yaml", "weir proxy config file path")
-	logEncoder := rootCmd.PersistentFlags().String("log_encoder", "", "log in format of tidb, console, or json")
+	logEncoder := rootCmd.PersistentFlags().String("log_encoder", "", "log in format of tidb, newtidb, console, or json")
```

Comment:
I've trust `log.NewTextEncoder` much that I just modified the encoder instead of writing a new one from the scratch.

Most bugs of `pingcap/log` is not in the encoder, but comes from its buggy API. Thus I did not even include one line.

Reply:
The modification of `encoder` is seen as a improvement that should not have problems. Or even if it has, like now, I can fix it easily.

Reply:
It is removed from the description.

Reply:
> Did you fix any bugs on your version?

I don't, neither the old encoder. The old encoder have not been modified since Mar 2019.